RELEASING.md

```diff
@@ -6,16 +6,18 @@ Releasing checklist:
 
 (2) Checkout the `main` branch. This is the only releasable branch.
 
-(3) Create a tag for the release.
+(3) Update the version in `docs/antora.yml`, create a PR, and wait for it to be merged.
```

Comment:
Then is not clear on which branch should `docs/antora.yml` be updated. Should we add another step for a branch?

Reply:
it's not relevant what branch is it, as long as it's not `main`. all PRs are made in separate branches, no?

Reply:
It could be main as well (or any other branch), as long as you are working on a fork, not directly on the repo (most of times for contributors). Even if can be any branch, I would specify a convention, because is easier for contributors to follow IMO, and they don't necessarily know it can be any branch (even when we do).

Reply:
technically anoter repo's `main` is still a different branch (usually `myrepo/main`). i don't think the branch is relevant nor it'll be confusing if we don't specify, as we don't do it for regular PRs.

finally, this is for releasing not for external contributors.

Reply:
That's fair, I will swap it with 2.